apps/dashboard/src/components/contract-functions/contract-function.tsx

```diff
@@ -20,7 +20,12 @@ import {
   Thead,
   Tr,
 } from "@chakra-ui/react";
-import type { Abi, AbiEvent, AbiFunction } from "abitype";
+import {
+  type Abi,
+  type AbiEvent,
+  type AbiFunction,
+  formatAbiItem,
```

Comment:
instead use `toFunctionSelector` to generate the deterministic function selector -> then you can map back from that into the selected function inside the explorer

Reply:
viem also exports `toFunctionSelector` and you can pass it any abiFunction